web/templates/settings/admin_emoji_list.hbs

```diff
@@ -12,7 +12,9 @@
     </td>
     <td>
         {{#if author}}
-        <span class="emoji_author">{{author.full_name}}</span>
+        {{#with author}}
+        <span class="emoji_author panel_user_list">{{> ../user_display_only_pill display_value=full_name img_src=avatar_url}}</span>
+        {{/with}}
```

Comment:
running `./tools/lint` command it is giving me error you cannot use else/else if tag without if tag.

Reply:
Is using `#with` tag necessary here? can't we just get `full_name` and `avatar_url` values from `author` object?

Reply:
Initially, I wasn't using the `with` tag. However, I believe @adnan-td is suggesting that if we don't utilize `with` `display_value=author.full_name img_src=author.avatar_url user_id=author.user_id is_active=author.is_active`, the code can become excessively lengthy and cluttered. One thing is we can do is define these variables in js code that way it will not be cluttered here which was the initially implementation.

Reply:
I think this can be somewhat confusing to pass some parameters directly and some using the `with` tag, but I am nto sure.